TODO.md

```diff
@@ -16,3 +15,4 @@ Bigger things
 - add grpc API explorer (https://github.com/grpc-ecosystem/awesome-grpc#gui)
 - add support for eclair, btcd
 - POC testing-library for testcafe (https://testing-library.com/docs/testcafe-testing-library/intro)
+- Upgrade dependencies versions
```

Comment:
What are you referring to here? The dependencies are automatically updated by the renovatebot.